utils/cli/tests/google/sass.test.ts

```diff
@@ -0,0 +1,34 @@
+import { describe, expect, it } from 'vitest';
+
+import { sassMetadata } from '../../src/sass/metadata';
+import { Metadata } from '../../src/types';
+import mockSassMetadata from './fixtures/sass-metadata.json';
+
+describe('sass', () => {
+	it('should generate Carlito sass metadata successfully', async () => {
```

Comment:
Probably rather than stating the font name in the test case, refer it as "no unicode range" or "variable" for those unfamiliar to the project.

Reply:
That's probably a good idea. I initially did that to keep it consistent with the other tests.

Reply:
Although, I was only looking at v1 and v2 packager tests. I didn't see the more helpful variable packager descriptions.

Reply:
I'll look into amending the other tests then. 👍